src/main/java/btb/constants/Constant.java

```diff
@@ -0,0 +1,22 @@
+package btb.constants;
+
+public abstract class Constant {
+
+    // declaration of final variables
+    public static final String DOTTED_LINE = "    --------------------------------------------------";
+
+    public static final String LOGO = "        ___     _       ___\n" +
+            "       | _ )   | |_    | _ )\n" +
+            "       | _ \\   |  _|   | _ \\\n" +
+            "       |___/   _\\__|   |___/\n" +
+            "     _|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"|\n" +
+            "     \"`-0-0-'\"`-0-0-'\"`-0-0-'";
+
+    public static final String BYE = "        ___                       _              ___     _  _\n" +
+            "       / __|    ___     ___    __| |     o O O  | _ )   | || |   ___\n" +
+            "      | (_ |   / _ \\   / _ \\  / _` |    o       | _ \\    \\_, |  / -_)\n" +
+            "       \\___|   \\___/   \\___/  \\__,_|   TS__[O]  |___/   _|__/   \\___|\n" +
+            "     _|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"| {======|_|\"\"\"\"\"|_| \"\"\"\"|_|\"\"\"\"\"|\n" +
+            "     \"`-0-0-'\"`-0-0-'\"`-0-0-'\"`-0-0-'./o--000'\"`-0-0-'\"`-0-0-'\"`-0-0-'";
+
+}
```

Comment:
Good job using constants to define some of your strings! You could extend this to all your other magic strings, numbers or even booleans :)